data/clusters/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
why creating a new .gitignore instead of using the one we've already got?

Reply:
Modification made in commit 540b521953feea4e73758f6d8173a1c58e9e65be

Reply:
The new .gitignore was created because I had a problem adding a empty folder in the repository (data/cluster), However this problem has been solved.